frontend/packages/data-portal/e2e/pageObjects/filters/filtersActor.ts

```diff
@@ -0,0 +1,170 @@
+import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
+import { translations } from 'e2e/constants'
+
+import { GetRunByIdQuery } from 'app/__generated__/graphql'
+import { QueryParams } from 'app/constants/query'
+import { getRunById } from 'app/graphql/getRunById.server'
+
+import { FiltersPage } from './filtersPage'
+import {
+  getAnnotationRowCountFromData,
+  getExpectedFilterCount,
+  getExpectedTotalCount,
+  getExpectedUrlWithQueryParams,
+} from './utils'
+
+export class FiltersActor {
```

Comment:
Could you add some documentation on what is expected to be in an Actor and what isn't?

Reply:
Done! Updated documentation is here: https://github.com/chanzuckerberg/cryoet-data-portal/wiki/Testing